src/MainModel.elm

```diff
@@ -42,6 +42,8 @@ type alias Model =
     , invalidSubmissionIdsInput : String
     , showValidation : Bool
     , scheduleSubmissionsIndividually : Bool
+    , showMobileView : Bool
+    , displayedColumns : List Column
```

Comment:
As far as I can tell we only use the id of the 1st column in this list. Do we need store a list of columns here or could it just be a `Maybe Int`? I understand if this due to planning for a future feature.

Reply:
done